selfdrive/modeld/SConscript

```diff
@@ -44,6 +44,10 @@ for pathdef, fn in {'TRANSFORM': 'transforms/transform.cl', 'LOADYUV': 'transfor
     xenv['CXXFLAGS'].append(f'-D{pathdef}_PATH=\\"{File(fn).abspath}\\"')
 
 # Compile cython
+snpe_rpath_qcom = "/data/pythonpath/third_party/snpe/larch64"
+snpe_rpath_pc = f"{Dir('#').abspath}/third_party/snpe/x86_64-linux-clang"
+snpe_rpath = lenvCython['RPATH'] + [snpe_rpath_qcom if arch == "larch64" else snpe_rpath_pc]
```

Comment:
We used to set LD_LIBRARY_PATH in the navmodeld launch script, but that's been removed since navmodeld is now a PythonProcess forked from manager.py. snperunner still needs to be linked to libSNPE.so, and after some research it _seems_ like the most straightforward way to do this is to set the RPATH in scons. However, I'm not super familiar with these linker flags and it's possible there are some downsides I'm not aware of. I also removed `/usr/lib/aarch64-linux-gnu` and `"#/openpilot/third_party/snpe/x86_64-linux-clang` for TICI/PC, respectively, since it doesn't seem like libSNPE ever exists in the former as far as I can tell, and the later path seems to not exist at all.

Reply:
nope, RPATH is good (even preferable over the wrapper scripts). we didn't use to do it because android (bionic) didn't support it